### PR TITLE
More homepage links

### PR DESCRIPTION
**Pull request by Jujhar**
Addresses #575 

Added four more links to footer (dcrtime, bounty, mainnet, testnet)

Screen Shot 2019-05-30 at 11 01 11 PM

### CODE REVIEW

File: src/layouts/partials/footer.html [hunk 1]
```diff
@@ -26,6 +28,8 @@
                 </div>
                 <div class="footerblockstatsrow w-clearfix"><a id="footerDownloads" class="backgroundwashedblue borderradius4 dropshadow footerblockstatslink" href="{{ "downloads/" | relLangURL }}">52k {{ T "general_total" }}</a>
                     <div class="footerblockstatsname">{{ T "general_downloads" }}</div>
+                    <a href="https://mainnet.decred.org/" target="_blank" class="footer-inline footerblocklink transition">{{ T "nav_mainnet" }}</a>
```
Comment (dajohi) [inline src/layouts/partials/footer.html]: change to dcrdata.decred.org

Comment (dajohi) [reply src/layouts/partials/footer.html]: all links missing rel="noopener noreferrer"

File: src/layouts/partials/footer.html [hunk 2]
```diff
@@ -5,7 +5,9 @@
     <div class="footercolumns width1100">
         <div class="footerrows w-row">
             <div class="footerrowscolumn1 w-col w-col-3 w-col-medium-3">
-                <a href="https://github.com/decred" target="_blank" rel="noopener noreferrer" class="footerblocklinkdev">{{ T "general_copyright" }}</a>
+                <a href="https://github.com/decred" target="_blank" rel="noopener noreferrer" class="footerblocklinkdev footer-margin-bottom">{{ T "general_copyright" }}</a>
+                <a href="https://time.decred.org" target="_blank" class="footer-inline footerblocklink transition footer-margin-top footer-margin-left">{{ T "nav_time" }}</a>
+                <a href="https://bounty.decred.org" target="_blank" class="footer-inline footerblocklink transition">{{ T "nav_bounty" }}</a>
             </div>
```
Comment (dajohi) [inline src/layouts/partials/footer.html]: what is the reasoning for adding time and bounty?

Comment (Jujhar) [reply src/layouts/partials/footer.html]: It does feel a little iffy to me maybe more development projects around 6 can be listed with it in a smaller font. Bounty makes sense there I think, as it let's people know it is available.

@zhizhongzhiwai was the one that suggested it on the task page.

Comment (dajohi) [reply src/layouts/partials/footer.html]: Use https://timestamp.decred.org instead of time.

Comment (Jujhar) [reply src/layouts/partials/footer.html]: Updated

**Comment by Jujhar** [comment]
Done

**Comment by dajohi** [comment]
@Jujhar please fix the conflicts

**Comment by Jujhar** [comment]
Fixed

File: transifex_catalogs/en.json [hunk 3]
```diff
@@ -10,6 +10,9 @@
   "description": "Decred is an autonomous digital currency. With a hybrid consensus system, it is built to be a decentralized, sustainable, and self-ruling currency where stakeholders make the rules.",
   "general_copyright": "Decred Developers",
   "nav_brand_resources": "Brand Resources",
+  "nav_bounty": "bounty",
+  "nav_dcrdata": "Block Explorer",
+  "nav_time": "dcrtime",
```
Comment (dajohi) [inline transifex_catalogs/en.json]: timestamp

**Pull request merged**